[32mINFO    [0m conda_build.metadata:metadata.py:854 Attempting to finalize metadata for conda-build-test-source-setup-py-data
[32mINFO    [0m conda_build.build:build.py:2292 Packaging conda-build-test-source-setup-py-data
[33mWARNING [0m conda_build.build:build.py:1627 No files or script found for output conda-build-test-source-setup-py-data
[33mWARNING [0m conda_build.build:build.py:1456 The install/build script(s) for conda-build-test-source-setup-py-data deleted the following files (from dependencies) from the prefix:
['lib/libffi.7.so']
This will cause the post-link checks to mis-report. Please try not to delete and files (DSOs in particular) from the prefix
[32mINFO    [0m conda_build.variants:variants.py:235 Adding in variants from /tmp/tmpoj665e90/info/recipe/conda_build_config.yaml
[32mINFO    [0m conda_build.utils:utils.py:2019 Renaming work directory '/tmp/cb/popen-gw0/test_recipe_builds_building_ji1/conda-build-test-source-setup-py-data_1669035373833/work' to '/tmp/cb/popen-gw0/test_recipe_builds_building_ji1/conda-build-test-source-setup-py-data_1669035373833/work_moved_conda-build-test-source-setup-py-data-1.21.0-0_linux-64_main_build_loop'
[32mINFO    [0m conda_build.utils:utils.py:2026 shutil.move(work)=/tmp/cb/popen-gw0/test_recipe_builds_building_ji1/conda-build-test-source-setup-py-data_1669035373833/work, dest=/tmp/cb/popen-gw0/test_recipe_builds_building_ji1/conda-build-test-source-setup-py-data_1669035373833/work_moved_conda-build-test-source-setup-py-data-1.21.0-0_linux-64_main_build_loop)